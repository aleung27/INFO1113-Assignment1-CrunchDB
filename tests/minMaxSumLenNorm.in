SET g 4 6 1 3
MIN g
MAX g
SUM g
LEN g
SET j -3333 0 10000000 2
MIN j
MAX j
SUM j
LEN j
BYE
