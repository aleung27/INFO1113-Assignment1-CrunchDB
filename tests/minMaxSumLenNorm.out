> ok

> 1

> 6

> 14

> 4

> ok

> -3333

> 10000000

> 9996669

> 4

> bye
